libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -353,6 +353,11 @@
 #define HAL_FLOW_PX4_MAX_FLOW_PIXEL 4
 #define HAL_FLOW_PX4_BOTTOM_FLOW_FEATURE_THRESHOLD 30
 #define HAL_FLOW_PX4_BOTTOM_FLOW_VALUE_THRESHOLD 5000
+#define HAL_GPIO_A_LED_PIN        69
+#define HAL_GPIO_B_LED_PIN        68
+#define HAL_GPIO_C_LED_PIN        45
```

Comment:
Ok.

I should change on buzzer too ?
# define BUZZER_PIN     11 // GPIO P8_32
to
 # define BUZZER_PIN     BBB_P8_25
?